.github/workflows/query-engine.yml

```diff
@@ -17,56 +17,57 @@ concurrency:
 
 jobs:
   rust-query-engine-tests:
-    name: "Test ${{ matrix.database.name }} (${{ matrix.engine_protocol }}) on Linux"
+    name: 'Test ${{ matrix.database.name }} (${{ matrix.engine_protocol }}) on Linux'
 
     strategy:
       fail-fast: false
       matrix:
         database:
-          - name: "vitess_5_7"
+          - name: 'vitess_5_7'
             single_threaded: true
-            connector: "vitess"
-            version: "5.7"
-          - name: "vitess_8_0"
+            connector: 'vitess'
+            version: '5.7'
+          - name: 'vitess_8_0'
             single_threaded: true
-            connector: "vitess"
-            version: "8.0"
-          - name: "postgres15"
+            connector: 'vitess'
+            version: '8.0'
+          # Why are we testing v15 and not older?
+          - name: 'postgres15'
             single_threaded: true
-            connector: "postgres"
-            version: "15"
-          - name: "mssql_2022"
+            connector: 'postgres'
+            version: '15'
```

Comment:
👀

Reply:
I asked in Slack, let's see https://prisma-company.slack.com/archives/C4GCG53BP/p1696585370897559